scripts/main.js

```diff
@@ -1 +1,117 @@
-// TODO: add JS
\ No newline at end of file
+// ------* selecting elements that will change with scene selection *--------
+const html = document.documentElement;
+const header = document.querySelector('h1');
+const select = document.querySelector('select');
+const audio = document.querySelector('audio');
+
+// ------* function declarations *---------
+
+function onSceneOptionSelected(event) {
+    const sceneSelection = event.target.value;
+    localStorage.setItem('selection', sceneSelection);
+    const selection = localStorage.getItem('selection');
+
+    const chooseSceneOption = document.querySelector('#choose');
+    if (chooseSceneOption) {
+        chooseSceneOption.remove();
+    }
+    
+    renderSelectedScene(selection);
+}
+
+function onClickToStartMedicationClicked() {
+    const overlay = document.querySelector('#overlay');
+    overlay.remove();
+}
+
+function renderSelectedScene(selection) {
+    renderBackground(selection);
+    renderHeader(selection)
+    renderSelectionBackground(selection);
+    renderAudio(selection);
+}
+
+function renderBackground(selection) {
+    if (selection === 'beach') {
```

Comment:
Definitely check against a constant instead of directly against a string. Checking against strings like this allows for hard to detect bugs just from a typo. Try defining constants at the top of the file